pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -664,6 +664,55 @@ public static class QueryOptionValue {
       }
     }
 
+    /**
+     * Calcite and Pinot rule names / descriptions
+     * used for enable and disabling of rules, this will be iterated through in PlannerContext
+     * to check if rule is disabled.
+     */
+    public static class PlannerRuleNames {
+      public static final String PINOT_FILTER_INTO_JOIN = "PinotFilterIntoJoinRule";
```

Comment:
Consider not adding `Pinot` in both variable name and value. We want to pick a descriptive name for usability